documentation/book/proc-metrics-deploying-prometheus-operator.adoc

```diff
@@ -0,0 +1,50 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-prometheus.adoc
+[id='proc-metrics-deploying-prometheus-operator-{context}']
+
+= Deploying the Prometheus Operator
+
+To deploy the Prometheus Operator to your Kafka cluster, apply the YAML resource files from the https://github.com/coreos/prometheus-operator/tree/master/example/rbac/prometheus-operator[Prometheus CoreOS repository].
+
+.Procedure
+
+. Download and edit the resource files from the repository to use a different `namespace`:
++
+On Linux, use:
++
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-deployment.yaml | sed -e 's/namespace: .\*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
++
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml > prometheus-operator-cluster-role.yaml
++
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role-binding.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-cluster-role-binding.yaml
++
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-service-account.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-service-account.yaml
++
+On MacOS, use:
++
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-deployment.yaml | sed -e '' 's/namespace: .\*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
++
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml > prometheus-operator-cluster-role.yaml
++
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role-binding.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-cluster-role-binding.yaml
++
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-service-account.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-service-account.yaml
++
+NOTE: If it is not required, you can manually remove the `spec.template.spec.securityContext` property from the `prometheus-operator-deployment.yaml` file.
+
+. Deploy the Prometheus Operator:
++
+[source,shell,subs="+quotes,attributes"]
```

Comment:
I don't think `+quotes,attributes` makes sense. It's saying quotes should be appended to the default substitutions, but that attributes replaces the default substitutions. IOW I think attributes itself should be preceded or followed by a `+`. See https://asciidoctor.org/docs/user-manual/#incremental-substitutions

Reply:
I think it works without the '+' on each property, but have added (and removed the unnecessary quotes)